app/src/main/res/layout/item_profile.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- https://qiita.com/ntsk/items/dac92596742e18470a55 -->
+<androidx.cardview.widget.CardView
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="?attr/selectableItemBackground"
```

Comment:
I'm not sure about what this means but Cirrus says there can be an overdraw, will this be a problem ?